uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/elasticsearch/suite/ElasticSearchTestSuite.java

```diff
@@ -72,8 +72,6 @@
                  "9300")
             .env("xpack.security.enabled",
                  "false")
-            .mountFrom(new File("src/test/resources/elasticsearch.yml").getAbsolutePath())
```

Comment:
Would it make sense to remove `src/test/resources/elasticsearch.yml` from resources (provided you're not using it anywhere else)?